apps/native-tests/ios/Podfile.lock

```diff
@@ -432,7 +432,7 @@ PODS:
     - Yoga
   - EXUpdatesInterface (2.0.0):
     - ExpoModulesCore
-  - FBLazyVector (0.82.0-rc.4)
+  - FBLazyVector (0.82.0-rc.5)
   - hermes-engine (0.82.0-rc.4):
```

Comment:
Should we update hermes engine here as well?

Reply:
good catch, yes we should